fsevents.js

```diff
@@ -6,6 +6,13 @@
 /* jshint node:true */
 'use strict';
 
+if(process.platform !== 'darwin') {
+  var err = new Error("Cannot find module 'fsevents'")
```

Comment:
Misleading error message, something along the lines of "fsevents is not compatible with your platform" is more accurate.

Style issue: space after `if` and please use single quotes.